pkg/ccl/backupccl/backup_test.go

```diff
@@ -97,6 +97,32 @@ func backupRestoreTestSetupEmptyWithParams(
 	return ctx, tc, sqlDB, cleanupFn
 }
 
+func createEmptyCluster(
```

Comment:
should this function be moved to full_cluster_backup_restore_test where it is used?

Reply:
Good point - I also moved the other cluster creation helper which is only used in full cluster tests.